### PR TITLE
Documentation: Using candi

### PR DESCRIPTION
**Pull request by ljhwang**
Adds year for clarity. Formats footnote. (and no commas!)

### CODE REVIEW

File: doc/sphinx/user/install/local-installation/using-candi.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ during the installation). We require at least the packages <span
 class="smallcaps">p4est</span>, TRILINOS, and
 finally DEAL.II.
 
-At the time of this writing `candi` will install <span
+At the time of this writing (2019), `candi` will install <span
```
Comment (MFraters) [inline doc/sphinx/user/install/local-installation/using-candi.md:14]: How do you get to the 2019? If I look back to the git blame the last real update to the versions was 10 months ago (https://github.com/geodynamics/aspect/blame/ce02c6677a17e5a2d449b81854a168c83b06f661/doc/manual/manual.tex#L3200).

Comment (ljhwang) [reply doc/sphinx/user/install/local-installation/using-candi.md:14]: Rene looked this up for me ...

Comment (tjhei) [reply doc/sphinx/user/install/local-installation/using-candi.md:14]: Can you change this to 2022 and "p4est 2.3.2, Trilinos 12.18.1, deal.II 9.3.3" ?

File: doc/sphinx/user/install/local-installation/using-candi.md [hunk 2]
```diff
@@ -11,7 +11,7 @@ during the installation). We require at least the packages <span
...
 class="smallcaps">p4est</span> 2.2, TRILINOS
 12.18.1, and DEAL.II 9.3.0. We strive to keep
```
Comment (gassmoeller) [inline doc/sphinx/user/install/local-installation/using-candi.md:16]:
```suggestion
At the time of this writing (2022), `candi` will install p4est 2.3.2, Trilinos
12.18.1, and deal.II 9.3.3. We strive to keep
```

**Review by gassmoeller — APPROVED** [review]

**Review by tjhei — APPROVED** [review]

**Comment by tjhei** [comment]
can you fix indentation?

File: doc/sphinx/user/install/local-installation/using-candi.md [hunk 3]
```diff
@@ -9,9 +9,9 @@ during the installation). We require at least the packages <span
 class="smallcaps">p4est</span>, Trilinos, and
 finally deal.II.
 
-At the time of this writing `candi` will install <span
-class="smallcaps">p4est</span> 2.2, Trilinos
-12.18.1, and deal.II 9.3.0. We strive to keep
+At the time of this writing (2022), `candi` will install 
+p4est 2.3.2, Trilinos 12.18.1, and deal.II 9.3.3. 
+We strive to keep
```
Comment (gassmoeller) [inline doc/sphinx/user/install/local-installation/using-candi.md:14]:
```suggestion
At the time of this writing (2022), `candi` will install
p4est 2.3.2, Trilinos 12.18.1, and deal.II 9.3.3.
We strive to keep
```

**Review by tjhei — APPROVED** [review]

**Pull request merged**